inspirehep/modules/disambiguation/ext.py

```diff
@@ -42,12 +42,12 @@ def init_config(self, app):
         disambiguation_base_path = os.path.join(app.instance_path, 'disambiguation')
 
         app.config['DISAMBIGUATION_BASE_PATH'] = disambiguation_base_path
-        app.config['DISAMBIGUATION_CLUSTERS_PATH'] = os.path.join(
-            disambiguation_base_path, 'clusters.json')
+        app.config['DISAMBIGUATION_CURATED_SIGNATURES_PATH'] = os.path.join(
+            disambiguation_base_path, 'signatures.jsonl')
+        app.config['DISAMBIGUATION_INITIAL_CLUSTERS_PATH'] = os.path.join(
```

Comment:
Now that I think about it, these should be the `INPUT_CLUSTERS`, since we are later going to add the `OUTPUT_CLUSTERS`...

Reply:
This also addresses this comment: https://github.com/inspirehep/inspire-next/pull/3449#discussion_r193377680.